### PR TITLE
Ft/bundlr fee support

### PR DESCRIPTION
**Pull request by fuadop**

### CODE REVIEW

File: src/commands/balance.ts [hunk 1]
```diff
@@ -24,8 +24,32 @@ const command: CommandInterface = {
...
+    try {
+      addy = await blockweave.wallets.jwkToAddress(wallet);
+    } catch (e) {
+      console.log(clc.red('Unable to decrypt wallet address.'));
+      if (debug) console.log(e);
+    }
+
+    if (useBundler) {
+      try {
+        const res = await bundler.get(`/account/balance?address=${addy}`);
```
Comment (cedriking) [inline src/commands/balance.ts:37]: I noticed that this one to get the address is being called here and https://github.com/textury/arkb/pull/35/files#diff-0cb87263fc2cd13d57e280a254b560b49292258ea6134b4d08b1ba2ebf1eea63R48 , let's best use a single call that will be available in the bundler class here: https://github.com/textury/arkb/blob/01b9fecba930b65c7d758e017a723aa2fed87053/src/utils/bundler.ts

**Review by cedriking — CHANGES_REQUESTED** [review]
Hey @fuadop I sent some information to review and update.

File: src/commands/fundBundler.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+      const tx = await blockweave.createTransaction(
+        {
+          target: bundlerAddress,
+          quantity: amount.toString(),
+        },
+        wallet,
+      );
+
+      tx.reward = parseInt(tx.reward, 10).toString();
```
[the remark this answers is not in the document]
Comment (fuadop) [reply src/commands/fundBundler.ts:63]: @cedriking what does a multiplier do ? it multiplies the reward ?

**Comment by fuadop** [comment]
Hello @cedriking ,

Can you please review now ?

File: src/commands/withdrawBundler.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import debugOption from '../options/debug';
+import helpOption from '../options/help';
+import timeoutOption from '../options/timeout';
+import { JWKInterface } from 'blockweave/dist/faces/lib/wallet';
+import { stringToBuffer } from 'blockweave/dist/utils/buffer';
+import { BundlerWithdraw } from '../faces/bundler';
+
+const command: CommandInterface = {
+  name: 'withdraw-bundler',
+  aliases: ['wb'],
```
Comment (cedriking) [inline src/commands/withdrawBundler.ts:16]: No alias for this one please

File: src/commands/fundBundler.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      clc.red('Error getting bundler address, see more info with the --debug option.');
+      if (debug) console.log(e);
+      process.exit(1);
+    }
+
+    // Fund the bundler address
+    try {
+      const addy = await blockweave.wallets.jwkToAddress(wallet);
+
+      const tx = await blockweave.createTransaction(
```
Comment (cedriking) [inline src/commands/fundBundler.ts:51]: We should use the Transfer class here: https://github.com/textury/arkb/blob/master/src/lib/transfer.ts

**Review by cedriking — CHANGES_REQUESTED** [review]
Only ones I've noticed that are missing is changing to the Transfer class, and removing the alias on withdraw-bundler. Thanks @fuadop !

**Comment by fuadop** [comment]
@cedriking

**Pull request merged**